markovflow/sde/sde_utils.py

```diff
@@ -18,22 +18,90 @@
 import tensorflow as tf
 
 from gpflow.base import TensorType
+from gpflow.quadrature import NDiagGHQuadrature
+from gpflow.probability_distributions import Gaussian
 
 from markovflow.sde import SDE
 from markovflow.state_space_model import StateSpaceModel
 
 
+class LinearDrift:
+    """
+    A linear drift of the form, f(x_t, t) = A_t * x_t + b_t.
+    """
+    def __init__(self, A: TensorType = None, b: TensorType = None):
+        self.A = A
```

Comment:
docstrings? Like say it takes two tensors as arguments

Reply:
Maybe overkill but some shape_asserts checking that the dims match (number of time steps would be good)

We do this when we create a ssm
https://github.com/secondmind-labs/markovflow/blob/654138d0dbb67dc6c3457eadf3e2db9cd8ffd71b/markovflow/state_space_model.py#L101

Reply:
The assert is there when these variables are used. As you mentioned, checking them multiple times would be an overkill.